srcpkgs/remake/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'remake'
+pkgname=remake
+version=4.2.1
+revision=1
+wrksrc=remake-4.2.1+dbg-1.5
```

Comment:
since the `dbg-1.5` suffix is used in a lot of places, add it as a variable and use it in other places.